airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -81,8 +81,13 @@ def Popen(self, cmd, **kwargs):
         """
         process = subprocess.Popen(cmd,
                                    stdout=subprocess.PIPE,
-                                   stderr=subprocess.PIPE,
+                                   stderr=subprocess.STDOUT,
                                    **kwargs)
+
+        for line in iter(process.stdout):
+            line = line.decode('utf-8').strip()
+            logging.info(line)
+
         output, stderr = process.communicate()
```

Comment:
If you're streaming stderr to stdout and reading everything directly from stdout, shouldn't `process.communicate()` be replaced by `process.wait()` and not capturing `output, stderr`?

Looking at the implementation of `process.communicate()` I think it might not add a lot here.